scripts/start-integration-env.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+BASEDIR=$(pwd)
+set -e
+sudo rm -rf ./build/networks
```

Comment:
I don't think sudo is needed. Running sudo prompts for a password which doesn't feel right from a dev experience perspective.

Reply:
removing sudo, at least on my linux build, results in permission denied on the crypto and will not delete the network folder